operator/src/test/java/oracle/kubernetes/operator/helpers/PodHelperTestBase.java

```diff
@@ -678,125 +677,129 @@ public void whenPodCreated_withNoPvc_fromModel_containerHasExpectedVolumeMounts(
+  public void whenDomainHasAuxiliaryImageAndVolumeWithCustomMountPath_createPodsWithVolumeMountHavingCustomMountPath() {
     getConfigurator()
-            .withCommonMountVolumes(getCommonMountVolume(CUSTOM_MOUNT_PATH))
-            .withCommonMounts(getCommonMounts("wdt-image:v1"));
+            .withAuxiliaryImageVolumes(getAuxiliaryImageVolume(CUSTOM_MOUNT_PATH))
+            .withAuxiliaryImages(getAuxiliaryImages("wdt-image:v1"));
 
     assertThat(getCreatedPodSpecContainers().get(0).getVolumeMounts(),
-            hasItem(new V1VolumeMount().name(getCommonMountVolumeName()).mountPath(CUSTOM_MOUNT_PATH)));
+            hasItem(new V1VolumeMount().name(getAuxiliaryImageVolumeName()).mountPath(CUSTOM_MOUNT_PATH)));
   }
 
   @Test
-  public void whenDomainHasCommonMountVolumeWithMedium_createPodsWithVolumeHavingSpecifiedMedium() {
+  public void whenDomainHasAuxiliaryImageVolumeWithMedium_createPodsWithVolumeHavingSpecifiedMedium() {
     getConfigurator()
-            .withCommonMountVolumes(Collections.singletonList(
-                    new CommonMountVolume().name(TEST_VOLUME_NAME).medium("Memory")))
-            .withCommonMounts(getCommonMounts("wdt-image:v1"));
+            .withAuxiliaryImageVolumes(Collections.singletonList(
+                    new AuxiliaryImageVolume().name(TEST_VOLUME_NAME).medium("Memory")))
+            .withAuxiliaryImages(getAuxiliaryImages("wdt-image:v1"));
 
     assertThat(getCreatedPod().getSpec().getVolumes(),
-            hasItem(new V1Volume().name(getCommonMountVolumeName()).emptyDir(
+            hasItem(new V1Volume().name(getAuxiliaryImageVolumeName()).emptyDir(
                     new V1EmptyDirVolumeSource().medium("Memory"))));
   }
 
   @Test
-  public void whenDomainHasCommonMountVolumeWithSizeLimit_createPodsWithVolumeHavingSpecifiedSizeLimit() {
+  public void whenDomainHasAuxiliaryImageVolumeWithSizeLimit_createPodsWithVolumeHavingSpecifiedSizeLimit() {
     getConfigurator()
-            .withCommonMountVolumes(Collections.singletonList(
-                    new CommonMountVolume().name(TEST_VOLUME_NAME).sizeLimit("100G")))
-            .withCommonMounts(getCommonMounts());
+            .withAuxiliaryImageVolumes(Collections.singletonList(
+                    new AuxiliaryImageVolume().name(TEST_VOLUME_NAME).sizeLimit("100G")))
+            .withAuxiliaryImages(getAuxiliaryImages());
 
     assertThat(getCreatedPod().getSpec().getVolumes(),
-            hasItem(new V1Volume().name(getCommonMountVolumeName()).emptyDir(
+            hasItem(new V1Volume().name(getAuxiliaryImageVolumeName()).emptyDir(
                     new V1EmptyDirVolumeSource().sizeLimit(Quantity.fromString("100G")))));
   }
 
   @Test
-  public void whenDomainHasCommonMountsWithImagePullPolicy_createPodsWithCMInitContainerHavingImagePullPolicy() {
+  public void whenDomainHasAuxiliaryImagesWithImagePullPolicy_createPodsWithCMInitContainerHavingImagePullPolicy() {
```

Comment:
Minor - method name can be changed to `whenDomainHasAuxiliaryImagesWithImagePullPolicy_createPodsWithAusInitContainerHavingImagePullPolicy`